superset/viz.py

```diff
@@ -327,8 +322,9 @@ def get_df_payload(self, query_obj=None):
         if query_obj and not is_loaded:
             try:
```

Comment:
The try/except block may no longer be required per the PR description. This would make the `stacktrace` obsolete as well.